Cue/app/tabs/library/CardListView.js

```diff
@@ -10,50 +11,107 @@ import EmptyView from '../../common/EmptyView'
 import CardListViewRow from './CardListViewRow'
 import ListViewHairlineSeparator from '../../common/ListViewHairlineSeparator'
 
+const styles = {
+  liftedRowStyle: {
+    opacity: 1,
+    transform: [{scale: 1.02}],
+    backgroundColor: 'white',
+    overflow: 'visible',
```

Comment:
What's the reason for this property? Is this for shadow or something? Because the content doesn't overflow out of the container, does it?